Storage/ThirdParty/SwiftData.swift

```diff
@@ -267,6 +267,8 @@ private class SQLiteDBStatement {
             } else if obj is Date {
                 let timestamp = (obj as! Date).timeIntervalSince1970
                 status = sqlite3_bind_double(pointer, Int32(index+1), timestamp)
+            } else if obj is UInt64 {
```

Comment:
> It strikes me as more Swifty to parse a UInt64 directly than to use an NSNumber to add to an Any in order to convert a UInt64 into a Double.

Yup I agree

> To be honest, I have no idea how this ever worked, if a previous version also passed in a UInt64.

So I think what was happening before was that the NSNumber gets interpreted as a Double which essentially captures anything a number could be (float, long long, etc). 

I'll make sure to go back to my previous PR and update NSNumber conversion to UInt64 when this lands!